toolkit/scripts/analysis.mk

```
@@ -71,6 +75,17 @@ sodiff-repo: $(SODIFF_REPO_FILE)
 
 $(SODIFF_REPO_FILE):
 	echo $(SODIFF_REPO_SOURCES) | sed -E 's:([^ ]+[.]repo):$(SPECS_DIR)/mariner-repos/\1:g' | xargs cat > $(SODIFF_REPO_FILE)
+	awk '{if ($$0 ~ /enabled=0/) $$0="enabled=1"; print}' $(SODIFF_REPO_FILE) > $(SODIFF_REPO_FILE)_fixed
+	mv $(SODIFF_REPO_FILE)_fixed $(SODIFF_REPO_FILE)
```

I may be misunderstanding the logic, but if we only want to make sure `enabled=0` is turned into `enabled=1` everywhere, then would this work to keep things simpler:
```suggestion
	sed -iE 's/enabled=0/enabled=1/g' $(SODIFF_REPO_FILE)
```

---

Having said that, it might be even cleaner and future-proof, if we enabled all repos when they are actually used. If we're using a package manager, then a single `--enablerepos=*` should do the trick. Unless there are some, you don't want to enable.

---

replaced the awk with sed for simplicity/constistency.